lsm-tree.cabal

```diff
@@ -77,46 +77,47 @@ description:
-  +----------+------------------------+-----------------+------------------------------------------------+
-  |          | Read next entry        | N\/A            | \(O(\frac{1}{P})\)                             |
-  +----------+------------------------+-----------------+------------------------------------------------+
+  The following table summarises the cost of the operations on LSM-trees measured in the number of disk I\/O operations.
+  If the cost depends on the merge policy or merge schedule, then the table contains one entry for each relevant combination.
+  Otherwise, the merge policy and\/or merge schedule is listed as N\/A.
+
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  | Resource | Operation              | Merge policy    | Merge schedule  | Cost in disk I\/O operations                   |
+  +==========+========================+=================+=================+================================================+
+  | Session  | Create\/Open           | N\/A            | N\/A            | \(O(1)\)                                       |
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  |          | Close                  | @LazyLevelling@ | N\/A            | \(O(o \: T \: \log_T \frac{n}{B})\)            |
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  | Table    | Create                 | N\/A            | N\/A            | \(O(1)\)                                       |
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  |          | Close                  | @LazyLevelling@ | N\/A            | \(O(T \: \log_T \frac{n}{B})\)                 |
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  |          | Lookup                 | @LazyLevelling@ | N\/A            | \(O(T \: \log_T \frac{n}{B})\)                 |
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  |          | Range Lookup           | @LazyLevelling@ | N\/A            | \(O(T \: \log_T \frac{n}{B} + \frac{b}{P})\)*  |
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  |          | Insert\/Delete\/Update | @LazyLevelling@ | @Incremental@   | \(O(\frac{1}{P} \: \log_T \frac{n}{B})\)       |
+  +----------+------------------------+-----------------+-----------------+------------------------------------------------+
+  |          |                        | @LazyLevelling@ | @OneShot@       | \(O(\frac{n}{P})\)                             |
```

Comment:
Could you elaborate on how you got to $O(\frac{n}{P})$? It's not clear to me that the $\log_T$ part would disappear when using the `OneShot` merge schedule